README.md

```diff
@@ -16,7 +16,7 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+Tags should be grouped by what they do (<footer><p></p></footer>) as opposed by how the programmer feels (<wallaby's store bottom><how to contact wallaby></wallaby's store bottom></how to contact wallaby>).
```

Comment:
You get bonus points for creativity! Semantic tags like `<p>` and `<header>` tell you the function of the text inside those tags. Non-semantic tags like `<b>` and `<i>` simply tell you what those tags should look like.